api/db/auth.js

```diff
@@ -89,7 +89,7 @@ const getUserAffiliatedStates = async (userId, { db = knex } = {}) =>
  * @returns {Object} { stateId: activities }
  */
 const getUserPermissionsForStates = async (userId, { db = knex } = {}) => {
-  const roles = await getRolesAndActivities();
+  const roles = (await getRolesAndActivities()) || [];
```

Comment:
I added the empty arrays because Knex returns undefined if nothing is found in a query. So if I return an empty array, then the method that is using it, populateUser, won't fail if the user doesn't have any affiliations.